auth/server/src/main/java/com/seagates3/authserver/AuthServerGetHandler.java

```diff
@@ -92,6 +92,15 @@ void run() {
       AuthServerConfig.setReqId(BinaryUtil.getAlphaNumericUUID());
     }
 
+    String stripped_request_id = AuthServerConfig.getReqId();
+    AuthServerConfig.setStripedReqId(stripped_request_id);

```

Comment:
add this as part of else condition in below code..

Reply:
This part is required to print the entire request Id the first time its encountered.
The second setstripedreqid is used to set the actual stripped request id.